lib/byron/src/Cardano/Wallet/Byron.hs

```diff
@@ -181,7 +181,7 @@ serveWallet
     -- ^ Which host to bind.
     -> Listen
     -- ^ HTTP API Server port.
-    -> AddrInfo
+    -> FilePath
```

Comment:
why reverting this to a 'FilePath' :thinking: ? `AddrInfo` is actually clearer about what this parameter is. A `FilePath` can be anything..